Dockerfile

```diff
@@ -5,7 +5,6 @@ RUN wget -qO- https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add
 RUN echo 'deb https://deb.nodesource.com/node_0.12 jessie main' > /etc/apt/sources.list.d/nodesource.list
 RUN apt-get update && apt-get install -y nodejs
 
-ENV GEM_HOME=/bundle
 RUN gem update --system
 RUN gem install bundler
```

Comment:
I was thinking it could be handy to get the latest gem and bundler for production images on build, but it looks like the 2.2.2 tag on Dockerhub is [regularly updated](https://github.com/docker-library/ruby/tree/master/2.2) with the latest.